projects/person-search-index-from-delius/container/Dockerfile

```
@@ -1,13 +1,31 @@
-FROM opensearchproject/logstash-oss-with-opensearch-output-plugin:7.16.3
+FROM opensearchproject/logstash-oss-with-opensearch-output-plugin:8.4.0
 
 COPY --chown=logstash ojdbc11.jar statement.sql /etc/logstash/
 COPY --chown=logstash logstash-full-load.conf /usr/share/logstash/full-load/logstash.conf
 COPY --chown=logstash logstash-full-load-dlq.conf /usr/share/logstash/full-load-dlq/logstash.conf
 COPY --chown=logstash logstash-incremental.conf /usr/share/logstash/incremental/logstash.conf
 COPY --chown=logstash logstash-incremental-dlq.conf /usr/share/logstash/incremental-dlq/logstash.conf
 COPY --chown=logstash logstash.yml pipelines.yml /usr/share/logstash/config/
+
+COPY --chown=logstash management/person-search-pipeline.json /tmp/init/
+COPY --chown=logstash management/person-search-template.json /tmp/init/
+COPY --chown=logstash management/init-script.sh /tmp/init/
+
+COPY --chown=logstash management/index-alias-management.sh /tmp/
+
+USER root
+RUN apt-get update && apt-get -y install cron && apt-get -y install jq
+RUN chmod u+s /usr/sbin/cron
```

We should clean up after apt to reduce the size of the image
```suggestion
RUN apt-get update && apt-get -y install \
  cron \
  jq \
  && rm -rf /var/lib/apt/lists/* \
  && chmod u+s /usr/sbin/cron
```